.tekton/pipeline-patch.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+for konflux_file in "${dir}"/gatekeeper-operator-fbc-*.yaml; do
+  echo "Updating $(basename "${konflux_file}") ..."
+
+  # Add hermetic build
+  has_hermetic=$(yq -o yaml '.spec.params | any_c(.name == "hermetic")' "${konflux_file}")
+  if [[ ${has_hermetic} == false ]]; then
+    yq '.spec.params |= . + [{"name":"hermetic","value":"true"}]' -i "${konflux_file}"
```

Comment:
I wonder if kustomize patches would be able to do this, as opposed to a script with `yq`. For reference: https://github.com/konflux-ci/build-definitions/blob/main/pipelines/fbc-builder/patch.yaml

But this is not a necessary change for this PR, just a thought for the future.

Reply:
Good point. A patch file like that would probably be nicer. 🤔 Though the limitation on patching arrays and needing to know the array index could make it difficult. But perhaps a strategic merge instead of JSON patch...